includes/controls/date-time.php

```diff
@@ -62,7 +62,7 @@ public function content_template() {
 		<div class="elementor-control-field">
 			<label for="<?php $this->print_control_uid(); ?>" class="elementor-control-title">{{{ data.label }}}</label>
 			<div class="elementor-control-input-wrapper">
-				<input id="<?php $this->print_control_uid(); ?>" placeholder="{{ data.placeholder }}" class="elementor-date-time-picker flatpickr" type="text" data-setting="{{ data.name }}">
+				<input id="<?php $this->print_control_uid(); ?>" placeholder="{{ elementor.getPanelView().currentPageView.children.findByModelCid( data._cid )?.getControlPlaceholder() }}" class="elementor-date-time-picker flatpickr" type="text" data-setting="{{ data.name }}">
```

Comment:
Find a shorter way or create a method for it.
@kobizz can you advise?

Reply:
You can pass the model as a template helper in `assets/dev/js/editor/controls/base.js` under `templateHelpers()` method, then you can simply call `{{ model.getControlPlaceholder() }}`